### ISSUE TITLE
Denided run app with C++ runner on Tizen 6

### ISSUE THREAD

**Issue by JIeoHugyC** [flutter-tizen/flutter-tizen#268]
Hi!
I tried to run demo flutter-tizen app on Tizen 6 TV with C++ runner and got an error about dangerous code. With C# runner all works.
Where can be problem?
![error](https://user-images.githubusercontent.com/34094611/140045471-c694c1c3-e24d-49df-b1da-46d44274c067.png)
..

**Comment by swift-kim** [comment]
Hi. The native runner (C++ language app) is not supported by Tizen 6.0 TVs as stated in [Supported commands](https://github.com/flutter-tizen/flutter-tizen/blob/master/doc/commands.md#create) and the command help (`flutter-tizen create -h`). Unlike native libraries (.so files), native **executables** cannot be executed on Tizen 6.0 TVs due to Samsung VD division's security policy.

**Comment by lenchis001** [comment]
Hi @swift-kim ! Do you know something about using C++/CLI for Tizen? We tried to build [this](https://github.com/mjrousos/CppCliMigrationSample) example for Tizen, but build fails: mismatch architecture of C++/CLI project and MSIL in main .NET project.

**Comment by swift-kim** [comment]
@lenchis001 As far as I know C++/CLI is Windows-only and cannot be ported to Tizen. (Oops. Why are you even trying to do it?) I don't know many details though.

**Comment by lenchis001** [comment]
@swift-kim As I understand from [this](https://devblogs.microsoft.com/cppblog/porting-a-c-cli-project-to-net-core/) post, this technology was ported to .NET Core. The problem is we have some C++ code, which we cannot be used with plugins system (it's a dynamic load library). We tried to rebuild that as a static library for C++ runner, but it failed because Tizen TV (even 6.0) disallows C++ runners. Now we are trying to reuse code of this library using C++/CLI technology, but we faced this the issue about mismatch architecture.
